dockers/docker-base-stretch/sources.list

```diff
@@ -1,8 +1,14 @@
 ## Debian mirror on Microsoft Azure
 ## Ref: http://debian-archive.trafficmanager.net/
 
-deb http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
-deb http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
-deb http://debian-archive.trafficmanager.net/debian/ stretch-backports main contrib non-free
+deb [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
+deb-src [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
+deb [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
+deb-src [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
+deb [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian/ stretch-backports main contrib non-free
+
+# ARM repo
+deb [arch=armhf,arm64] http://deb.debian.org/debian stretch main contrib non-free
+deb-src [arch=armhf,arm64] http://deb.debian.org/debian stretch main contrib non-free
+deb [arch=armhf,arm64] http://security.debian.org stretch/updates main contrib non-free
+deb-src [arch=armhf,arm64] http://security.debian.org stretch/updates main contrib non-free
```

Comment:
Image size increase is due to adding arm repo list in apt sources.

In base line sonic build:
Total repos 20
Fetched 15.7 MB i

In this PR:
Total repos 51
Fetched 46.4 MB

Reply:
can you fix this?

Reply:
Sure, will do it